stdlib/format.ml

```diff
@@ -838,6 +838,14 @@ let pp_safe_set_geometry state ~max_indent ~margin =
 let pp_get_geometry state () =
   { margin = pp_get_margin state (); max_indent = pp_get_max_indent state () }
 
+let pp_set_full_geometry state {margin; max_indent} =
+  pp_set_geometry state ~max_indent ~margin;
+  ()
```

Comment:
nit: surrounding code doesn't seem to use the trailing `()` style

Reply:
Ah, I understand now why I used this style. My intention was to define `pp_set_full_geometry` as a sequence of simpler `pp_set` functions, and then redefine `pp_set_geometry` by using it, instead of the other way around. This is done in the third commit of the patchset, and there I think the trailing `()` makes perfect sense (we will add more lines before as we add more geometry fields, and having consistent line endings give nicer diffs), but here it is not useful yet, I will get rid of it. Thanks.